tensilelite/Tensile/SolutionStructs.py

```diff
@@ -2180,6 +2180,8 @@ def assignDerivedParameters(state):
           break
         else:
           optVW //= 2
+      else:
+        state["VectorWidthA"] = 1
```

Comment:
This while-else will not happen because of "while 1"

Reply:
update